scripts/trento-support.sh

```diff
@@ -34,18 +34,18 @@ collect_base_system() {
     echo "#==[ Command ]======================================#"
     echo "# $(which helm) get hooks trento-server"
     helm get hooks trento-server
-    
+
```

Comment:
We need to remove the `.data.privatekey` here I think. In line 40

Reply:
We should remove the `ACCESS_TOKEN_ENC_SECRET` and related variables.
And maybe the rabbitmq username and password.